v2.1/configure-replication-zones.md

```diff
@@ -105,425 +95,58 @@ Constraints can be specified such that they apply to all replicas in a zone or s
 
-Currently, only members of the `admin` role can configure replication zones. By default, the `root` user belongs to the `admin` role.
-
-### Logging
+## Manage replication zones
 
-By default, the `zone` command logs errors to `stderr`.
-
-If you need to troubleshoot this command's behavior, you can change its [logging behavior](debug-and-error-logs.html).
+Use the [`CONFIGURE ZONE`](configure-zone.html) statement to add, modify, reset, and remove replication zones.
 
 ## Basic examples
 
-These examples focus on the basic approach and syntax for working with zone configuration. For examples demonstrating how to use constraints, see [Scenario-based Examples](#scenario-based-examples).
+These examples focus on the basic approach and syntax for working with zone configuration. For examples demonstrating how to use constraints, see [Scenario-based examples](#scenario-based-examples).
 
-###  List the pre-configured replication zones
+More examples are also on the [`CONFIGURE ZONE`](configure-zone.html) and [`SHOW ZONE CONFIGURATIONS`](show-zone-configurations.html) pages.
 
-Newly created CockroachDB clusters start with some special pre-configured replication zones:
+### View all replication zones
 
-{% include copy-clipboard.html %}
-~~~ shell
-$ cockroach zone ls --insecure --host=<node address>
-~~~
+{% include v2.1/zone-configs/view-all-replication-zones.md %}
 
-~~~
-.default
-.liveness
-.meta
-system.jobs
-~~~
+### View the default replication zone
 
-###  View the default replication zone
-
-The cluster-wide replication zone (`.default`) is initially set to replicate data to any three nodes in your cluster, with ranges in each replica splitting once they get larger than 67108864 bytes.
-
-To view the default replication zone, use the `cockroach zone get .default` command with appropriate flags:
-
-{% include copy-clipboard.html %}
-~~~ shell
-$ cockroach zone get .default --insecure --host=<node address>
-~~~
-
-~~~
-.default
-range_min_bytes: 1048576
-range_max_bytes: 67108864
-gc:
-  ttlseconds: 86400
-num_replicas: 3
-constraints: []
-~~~
+{% include v2.1/zone-configs/view-the-default-replication-zone.md %}
 
 ### Edit the default replication zone
 
-{{site.data.alerts.callout_danger}}
-{% include {{page.version.version}}/known-limitations/system-range-replication.md %}
-{{site.data.alerts.end}}
-
-To edit the default replication zone, create a YAML file defining only the values you want to change (other values will be copied from the `.default` zone), and use the `cockroach zone set .default -f <file.yaml>` command with appropriate flags:
-
-{% include copy-clipboard.html %}
-~~~ shell
-$ cat default_update.yaml
-~~~
-
-~~~
-num_replicas: 5
-~~~
-
-{% include copy-clipboard.html %}
-~~~ shell
-$ cockroach zone set .default \
---insecure \
---host=<node address> \
--f default_update.yaml
-~~~
+{% include v2.1/zone-configs/edit-the-default-replication-zone.md %}
```

Comment:
Similar comments as elsewhere re: the warnings about configuring zones for these system ranges (in particular `.liveness` and `.system`, but again I don't really know what I'm talking about here, it's more of a prompt to clarify with engineering).